src/main/kotlin/uk/gov/dluhc/printapi/mapper/CertificateToPrintRequestMapper.kt

```diff
@@ -23,6 +23,8 @@ abstract class CertificateToPrintRequestMapper {
     @Mapping(source = "printRequest.delivery.address.locality", target = "deliveryLocality")
     @Mapping(source = "printRequest.delivery.address.area", target = "deliveryArea")
     @Mapping(source = "printRequest.delivery.address.postcode", target = "deliveryPostcode")
+    @Mapping(expression = "java( uk.gov.dluhc.printapi.printprovider.models.PrintRequest.CertificateLanguage.EN )", target = "certificateLanguage")
+    @Mapping(expression = "java( uk.gov.dluhc.printapi.printprovider.models.PrintRequest.CertificateFormat.STANDARD )", target = "certificateFormat")
```

Comment:
This mapping line will change in the next PR, to map from the db entity field into this field